### PR TITLE
Fix errors for unit test on ARM64

### PR DESCRIPTION
**Pull request by zhlhahaha**
Fix errors for unit test on ARM64
1. seccomp test
2. TestUserLog

### CODE REVIEW

**Comment by zhlhahaha** [comment]
cc @lubinszARM

File: runsc/container/container_test.go [hunk 1]
```diff
@@ -1843,8 +1843,8 @@ func TestUserLog(t *testing.T) {
 		t.Fatal("error finding test_app:", err)
 	}
 
-	// sched_rr_get_interval = 148 - not implemented in gvisor.
-	spec := testutil.NewSpecWithArgs(app, "syscall", "--syscall=148")
+        // sched_rr_get_interval - not implemented in gvisor.
+        spec := testutil.NewSpecWithArgs(app, "syscall", "--syscall="+sched_rr_get_interval_sysno)
```
Comment (avagin) [inline runsc/container/container_test.go:1847]: can we use syscall.SYS_SCHED_RR_GET_INTERVAL?
https://golang.org/pkg/syscall/

Comment (zhlhahaha) [reply runsc/container/container_test.go:1847]: > can we use syscall.SYS_SCHED_RR_GET_INTERVAL?
> https://golang.org/pkg/syscall/

Thanks, I will give it a try.

Comment (zhlhahaha) [reply runsc/container/container_test.go]: > can we use syscall.SYS_SCHED_RR_GET_INTERVAL?
> https://golang.org/pkg/syscall/

It is a way better solution, thanks Andrei, patch updated.

**Pull request merged**